chia/_tests/pools/test_pool_cmdline.py

```diff
@@ -1,20 +1,1050 @@
+                    },
+                },
+            )
+        ]
+    )
+
+
+@pytest.mark.parametrize(
+    "wallet_environments",
+    [
+        {
+            "num_environments": 1,
+            "blocks_needed": [10],
+            "trusted": True,
+            "reuse_puzhash": False,
+        }
+    ],
+    indirect=True,
+)
+@pytest.mark.anyio
+async def test_plotnft_cli_inspect(
+    wallet_environments: WalletTestFramework,
+    capsys: pytest.CaptureFixture[str],
+) -> None:
+    wallet_state_manager: WalletStateManager = wallet_environments.environments[0].wallet_state_manager
+    wallet_rpc: WalletRpcClient = wallet_environments.environments[0].rpc_client
+    client_info: WalletClientInfo = WalletClientInfo(
+        wallet_rpc,
+        wallet_state_manager.root_pubkey.get_fingerprint(),
+        wallet_state_manager.config,
+    )
+
+    with pytest.raises(CliRpcConnectionError, match="No pool wallet found"):
+        await InspectPlotNFTCMD(
+            rpc_info=NeedsWalletRPC(
+                client_info=client_info,
+            ),
+            id=None,
+        ).run()
+
+    with pytest.raises(CliRpcConnectionError, match="is not a pool wallet"):
+        await InspectPlotNFTCMD(
+            rpc_info=NeedsWalletRPC(
+                client_info=client_info,
+            ),
+            id=15,
+        ).run()
+
+    wallet_id = await create_new_plotnft(wallet_environments)
+
+    # need to capture the output and verify
+    await InspectPlotNFTCMD(
+        rpc_info=NeedsWalletRPC(
+            client_info=client_info,
+        ),
+        id=wallet_id,
+    ).run()
+    out, _err = capsys.readouterr()
+    json_output = json.loads(out)
+
+    assert (
+        json_output["pool_wallet_info"]["current"]["owner_pubkey"]
+        == "0xb286bbf7a10fa058d2a2a758921377ef00bb7f8143e1bd40dd195ae918dbef42cfc481140f01b9eae13b430a0c8fe304"
+    )
+    assert json_output["pool_wallet_info"]["current"]["state"] == PoolSingletonState.FARMING_TO_POOL.value
+
+    wallet_id = await create_new_plotnft(wallet_environments, self_pool=True, second_nft=True)
+
+    with pytest.raises(CliRpcConnectionError, match="More than one pool wallet"):
+        await InspectPlotNFTCMD(
+            rpc_info=NeedsWalletRPC(
+                client_info=client_info,
+            ),
+            id=None,
+        ).run()
+
+    await InspectPlotNFTCMD(
+        rpc_info=NeedsWalletRPC(
+            client_info=client_info,
+        ),
+        id=wallet_id,
+    ).run()
+    out, _err = capsys.readouterr()
+    json_output = json.loads(out)
+
+    assert (
+        json_output["pool_wallet_info"]["current"]["owner_pubkey"]
+        == "0x893474c97d04a0283483ba1af9e070768dff9e9a83d9ae2cf00a34be96ca29aec387dfb7474f2548d777000e5463f602"
+    )
+
+    assert json_output["pool_wallet_info"]["current"]["state"] == PoolSingletonState.SELF_POOLING.value
+
+
+@pytest.mark.limit_consensus_modes(reason="unneeded")
```

Comment:
You can probably throw this decorator on most of the tests in here.  I usually go for `irrelevant` if you want consistency.